designsafe/static/scripts/data-depot/components/projects/pipeline-start/pipeline-start.template.html

```diff
@@ -93,4 +93,4 @@ <h3>Add/Remove Authors</h3>
             </div>
         </div>
     </div>
-</div>
+</div>
```

Comment:
This code is currently failing the CI pipeline because of a formatting issue. You can run the formatter in vscode by pressing cmd+shift+p and typing "format". The formatting tool should show up as the first option.